Description: Skip tests that are failing with fixtures.timeout.TimeoutException
             until bug is resolved.
Author: Corey Bryant <corey.bryant@canonical.com>
Bug: https://bugs.launchpad.net/cinder/+bug/1582370

--- a/cinder/tests/unit/test_coho.py
+++ b/cinder/tests/unit/test_coho.py
@@ -253,6 +253,8 @@
             [mock.call(INVALID_SNAPSHOT['volume_id'])])
 
     def test_snapshot_failure_when_remote_is_unreachable(self):
+        self.skipTest('Disabled on Ubuntu due to bug 1582370')
+
         drv = coho.CohoDriver(configuration=self.configuration)
 
         mock_get_volume_location = self.mock_object(coho.CohoDriver,
--- a/cinder/tests/unit/test_migrations.py
+++ b/cinder/tests/unit/test_migrations.py
@@ -810,6 +810,7 @@
         self.assertIsNotNone(fkey)
 
     def test_walk_versions(self):
+        self.skipTest('Disabled on Ubuntu due to bug 1582370')
         self.walk_versions(False, False)
 
 
@@ -828,6 +829,8 @@
         # add this to the global lists to make reset work with it, it's removed
         # automatically in tearDown so no need to clean it up here.
         # sanity check
+        self.skipTest('Disabled on Ubuntu due to bug 1582370')
+
         migration.db_sync(engine=self.migrate_engine)
 
         total = self.migrate_engine.execute(
